src/main/java/com/google/javascript/gents/ModuleConversionPass.java

```diff
@@ -515,13 +515,10 @@ void convertDestructuringRequireToImportStatements(Node n, ModuleImport moduleIm
     }
     // For the rest of the function, the imported and importing files are migrating together
 
-    // import {value as localName} from "./file"
+    // import {localName} from "./file"
     Node importSpec = new Node(Token.IMPORT_SPEC);
-    // import {a as b} only when a != b
     for (String localName : moduleImport.localNames) {
-      if (!moduleImport.moduleSuffix.equals(localName)) {
-        importSpec.addChildToBack(IR.name(localName));
-      }
+      importSpec.addChildToBack(IR.name(localName));
```

Comment:
This still looks very odd to me, because we don't do anything for renaming imports.